src/features/smartSearch/components/SmartSearchDialog/FilterGallery/filterGalleryPattern.ts

```diff
@@ -0,0 +1,107 @@
+import { FILTER_TYPE } from '../../types';
+
+const PATTERN_TEMPLATES = {
```

Comment:
Just to double check. What is the license for these patterns? Do we need to include a credit or license text here, maybe?

Reply:
I don't find any license info on the website and it doesn't say anything that i can find about usage... so... free...? https://www.cssportal.com/css-pattern-generator/

Reply:
"You will find hundreds of pages of tools, resources and generators to help with CSS and HTML. All of our generators will produce the necessary code that you can then copy and paste into your web project. Hopefully you find this website helpful in assisting you with your CSS and HTML needs."

Just copy, paste and go it sounds like to me

Reply:
Yeah, I agree, but I'm surprised there is no clear license anywhere. Maybe you can contact them just to make sure?

Reply:
I sent in a question via thier contact form, but it says that they're not able to respond to all emails heh, so we'll see if we get a response i guess

Reply:
"Thank you for your email. You are free to use the patterns how you wish, no credit is needed." yey :)